CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ All notable changes to the Wazuh app project will be documented in this file.
 - Fixed Wazuh menu and agent menu for Solaris agents [#2773](https://github.com/wazuh/wazuh-kibana-app/issues/2773) [#2725](https://github.com/wazuh/wazuh-kibana-app/issues/2725)
 - Fixed wrong shards and replicas for statistics indices and also fixed wrong prefix for monitoring indices [#2732](https://github.com/wazuh/wazuh-kibana-app/issues/2732)
 - Report's creation dates set to 1970-01-01T00:00:00.000Z [#2772](https://github.com/wazuh/wazuh-kibana-app/issues/2772)
+- Fixed bug for missing commands in ubuntu/debian and centos [#2725](https://github.com/wazuh/wazuh-kibana-app/issues/2786)
```

Comment:
The link text should be `#2786` and not `#2725` in:
```
[#2725](https://github.com/wazuh/wazuh-kibana-app/issues/2786)
```

Reply:
It's ok the first part [title] [issue id], #2725 is the issue that was fixed, but in the link should be the url to the same issue id #2725